README.md

```diff
@@ -1,24 +1,31 @@
 # deck.gl-native Dependencies
 
-This repository contains prebuilt dependencies for [deck.gl-native](https://github.com/UnfoldedInc/deck.gl-native). Libarries that are not available with Apt (Debian) or Homebrew are included here.
+This repository contains prebuilt dependencies for [deck.gl-native](https://github.com/UnfoldedInc/deck.gl-native).
 
 Supported architectures:
 
-| CPU | Operating System
-| --- | ----------------
-| x64 | Mac OSX
-| x64 | Linux (Ubuntu Bionic)
+| CPU    	| Operating System      	| Notes          	|
+|--------	|-----------------------	|----------------	|
+| x64    	| Mac OSX               	|                	|
+| x64    	| Linux (Ubuntu Bionic) 	|                	|
+| armv7  	| iOS                   	| Built Manually 	|
+| armv7s 	| iOS                   	| Built Manually 	|
+| arm64  	| iOS                   	| Built Manually 	|
+| arm64e 	| iOS                   	| Built Manually 	|
+| x86_64 	| iOS                   	| Built Manually 	|
```

Comment:
nit: I would standardize how we write x64/x86_64/amd64 in this table, just to make scanning it easier

Reply:
I was on the fence about this one, as standardizing these may introduce some confusion as it wouldn't match parameters specified when building appropriate architectures (`vcpkg` vs `cmake`)